HVSample/src/main/res/layout/activity_actionplan.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
if we only have one element (listview) i don't think we need to have a linear layout container? The listview can just be the container. If for some reason I am wrong in thinking that, at least change the root container to a framelayout since it is less expensive/complicated than a linear layout and we only have 1 child. But try just a listview first